problem2.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+
+
+fibonacci = np.zeros(33)
+fibevens = np.zeros(33)
+
+pos = 2
+fibonacci[0] = 1
```

Comment:
Do you need to store these numbers?  Can you think of a way of avoiding your second for loop here, and doing the sum in the first for loop?  And therefore, not storing all the Fibonacci numbers, just the keep the ones you're using?